src/main/java/seedu/command/AddExpenseCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.command;
+
+import seedu.category.Category;
+import seedu.transaction.Expense;
+import seedu.transaction.Transaction;
+import seedu.transaction.TransactionList;
+import java.util.List;
+
+public class AddExpenseCommand extends AddTransactionCommand {
+    public static final String COMMAND_WORD = "add-expense";
+    public static final String COMMAND_GUIDE = "add-expense [DESCRIPTION] [a/ AMOUNT] [d/ DATE] [c/ hCATEGORY]";
```

Comment:
As per our user guide, only the extra arguments need to be put in [ ]
You may try to change the guide to: "add-expense [DESCRIPTION] a/ AMOUNT [d/ DATE] [c/ CATEGORY]"